src/main/resources/com/google/api/codegen/java/build_gapic.gradle.snip

```diff
@@ -21,7 +21,9 @@
     compile 'com.google.api:gax:{@metadata.gaxVersionBound.lower}'
     compile 'com.google.api:gax-grpc:{@metadata.gaxGrpcVersionBound.lower}'
     testCompile 'junit:junit:4.12'
+    testCompile 'commons-lang:commons-lang:2.6'
     testCompile 'com.google.api:gax-grpc:{@metadata.gaxGrpcVersionBound.lower}:testlib'
+    testCompile 'io.grpc:grpc-netty-shaded:{@metadata.grpcVersionBound.lower}'
```

Comment:
Why is this necessary?

Reply:
I think we need this or the right combination of  (grpc-netty + netty-tcnative-boringssl-static) to send real requests. I figured using grpc-netty-shaded is a lot easier to manage. But you probably have more insights here.